### PR TITLE
Updating the builder's section of the wiki

### PR DESCRIPTION
**Pull request by lsaether**

### CODE REVIEW

**Comment by keorn** [comment]
This mostly seems to delete one major file.

**Review by keorn — DISMISSED** [review]
Looks good, but where is "build-with-polkadot" being moved?

**Comment by lsaether** [comment]
@keorn It's getting condensed to better reflect the format of the wiki. For example, the bulk of the section on acquiring parachain slots is better presented in the `candle.md` page so just links there.

File: docs/polkadot/learn/parachains.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ By connecting to Polkadot a parachain gains two major advantages over an indepen
 
 Parachains are operated and new blocks are authored by [collator nodes](../node/collator.md) that maintain "full-nodes" of a particular parachain. They retain all necessary information to be able to propose new blocks and execute transactions in a similar way as miners do in traditional PoW blockchains. A block proposed by a collator node will be checked by the validator node before it is accepted as a canonical state transition by the relay chain.
 
+## What is "parachain consensus"
```
Comment (laboon) [inline docs/polkadot/learn/parachains.md]: This should probably be "What is parachain consensus"? (question mark at end)

**Comment by laboon** [comment]
One very minor issue (above).  I'd just approve and trust you to fix it but I think someone would have to re-review anyways if you add a new commit.

**Comment by laboon** [comment]
@lsaether - Can you resolve the conflict as well?

**Review by laboon — APPROVED** [review]
👍

**Pull request merged**